### ISSUE TITLE
Finetune on VGG model

### ISSUE THREAD

**Issue by jjzhang10** [ZheC/Realtime_Multi-Person_Pose_Estimation#83]
I find the preprocessing step used in your code is different with that in VGG.
In your code:
      transformed_data[0*offset + i*img_aug.cols + j] = (rgb[0] - 128)/256.0;
      transformed_data[1*offset + i*img_aug.cols + j] = (rgb[1] - 128)/256.0;
      transformed_data[2*offset + i*img_aug.cols + j] = (rgb[2] - 128)/256.0;

In VGG: "In the paper, the model is denoted as the configuration E trained with scale jittering. The input images should be zero-centered by mean pixel (rather than mean image) subtraction. Namely, the following BGR values should be subtracted: [103.939, 116.779, 123.68]."

Does it matter for the performance? 
Or It may not matter, because the last few "fc" layers (shown as conv layers) will be retrained from the scratch.
Thanks.

**Comment by ildoonet** [comment]
I think this should be changed to the way of vgg model.

(But eventually, the model can be trained without modification, since finetuning process backpropagate gradients from top to bottom)
